app/src/main/java/org/fossasia/openevent/general/event/EventDetailsViewModel.kt

```diff
@@ -241,6 +246,16 @@ class EventDetailsViewModel(
             })
     }
 
+    fun loadOrders() {
+        compositeDisposable += orderService.getOrdersOfUser(getId())
```

Comment:
I think there should be a check statement if user is loggedIn() or not in here so that we won't waste a network call